blaze-client/src/main/scala/org/http4s/blaze/client/Http1Support.scala

```diff
@@ -76,13 +76,17 @@ private final class Http1Support[F[_]](
     connectTimeout,
   )
 
+  /* This is an uncancelable operation, and it's the caller's
+   * responsibility to guarantee this connection gets closed.
+   */
   def makeClient(requestKey: RequestKey): F[BlazeConnection[F]] =
     getAddress(requestKey) match {
       case Right(a) =>
         fromFutureNoShift(
           executionContextConfig.getExecutionContext.flatMap(ec =>
             F.delay(buildPipeline(requestKey, a, ec))
-          )
+          ),
+          None,
```

Comment:
I'd like to make this method return a resource, but it goes straight into the pool, and refactoring that will be a rippling change.